docs/commands.md

```diff
@@ -102,4 +102,79 @@ can be used to hint the user e.g. where to run `init` or `validate` from.
 }
 ```
 
+### `module.calls`
+
+List of module which are called from the current module.
+
+**Arguments:**
+
+ - `uri` - URI of the directory of the module in question, e.g. `file:///path/to/network`
+
+**Outputs:**
+
+ - `v` - describes version of the format; Will be used in the future to communicate format changes.
+ - `module_calls` - array of modules which are called from the module in question
+   - `name` - a unique identifier for this particular module
+   - `source_addr` - the source address given for this module in configuration
+   - `version` - the exact version of the module
```

Comment:
```suggestion
   - `version` - version constraint of the module call; applicable to modules hosted by the Terraform Registry (e.g. `~> 1.0`
```

Reply:
In retrospect I guess we should've called this `version_constraint`, but that's totally unrelated to this docs PR 😄